common/messages.proto

```diff
@@ -0,0 +1,55 @@
+// sets the .proto file syntax version
```

Comment:
device.proto more accurate?

Reply:
it's not just devices (eg. there is also healtcheck).
otherwise we should split it up. For now I will keep it like this.